src/core/server/index.ts

```diff
@@ -64,24 +64,14 @@ import type {
   SavedObjectsServiceSetup,
   SavedObjectsServiceStart,
 } from '@kbn/core-saved-objects-server';
-import { DeprecationsServiceSetup } from '@kbn/core-deprecations-server';
+import type { DeprecationsServiceSetup } from '@kbn/core-deprecations-server';
+import type { CoreUsageDataStart, CoreUsageDataSetup } from '@kbn/core-usage-data-server';
 
 import { HttpResources } from './http_resources';
 import { PluginsServiceSetup, PluginsServiceStart, PluginOpaqueId } from './plugins';
 import { UiSettingsServiceSetup, UiSettingsServiceStart } from './ui_settings';
 import { StatusServiceSetup } from './status';
-import { CoreUsageDataStart, CoreUsageDataSetup } from './core_usage_data';
 import { I18nServiceSetup } from './i18n';
-// Because of #79265 we need to explicitly import, then export these types for
-// scripts/telemetry_check.js to work as expected
-import {
-  CoreUsageStats,
```

Comment:
No longer required given we had to duplicate the types for https://github.com/elastic/kibana/issues/139389